src/App.jsx

```diff
@@ -0,0 +1,20 @@
+import "./App.css";
+import { Routes, Route } from "react-router-dom";
+import { Authentication, Home, WatchHistory, Playlist, LikedVideos, WatchLater, Profile} from "./pages";
+import Mockman from "mockman-js";
+function App() {
```

Comment:
All other components are arrow function but this is a normal function, you can change this as well to arrow function just for consistency